### PR TITLE
[FIX] project: align the div to the right in the activity view

### PR DESCRIPTION
**Pull request by hap-odoo**
The purpose of this commit is to display the project name and task
to the right of the div in the activity view.

before this commit, the project name and task displays next to
the many2many_tags div in the activity view.

TaskID-2646204

### CODE REVIEW

File: addons/project/views/project_views.xml [hunk 1]
```diff
@@ -1213,9 +1213,13 @@
                     <templates>
                         <div t-name="activity-box">
                             <field name="user_ids" widget="many2many_avatar_user"/>
-                            <div>
-                                <field name="name" display="full"/>
-                                <field name="project_id" muted="1" display="full" invisible="context.get('default_project_id', False)"/>
+                            <div class="ml-auto w-50">
```
[the remark this answers is not in the document]
Comment (xavierbol) [reply addons/project/views/project_views.xml:1216]: We could align the text to the right for this div.
```suggestion
                            <div class="text-right">
```

**Review by xavierbol — CHANGES_REQUESTED** [review]
Thanks for your work, I have made some comments and suggestions. 🙂

File: addons/project/views/project_views.xml [hunk 2]
```diff
@@ -1213,9 +1213,13 @@
                     <templates>
                         <div t-name="activity-box">
```
[the remark this answers is not in the document]
Comment (mba-odoo) [reply addons/project/views/project_views.xml]: @xavierbol  Done!

**Review by xavierbol — APPROVED** [review]
LGTM, thanks ! 🙂

**Review by IT-Ideas — APPROVED** [review]
LGTM Thanks! 🙂

**Review by IT-Ideas — APPROVED** [review]
LGTM Thanks 🙂 !

**Comment by IT-Ideas** [comment]
@robodoo r+

**Pull request closed without merging**